src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -192,9 +192,9 @@ private void buildServiceApplicationTab(ConfigBuilder<CaseData, State, UserRole>
     private void buildConditionalOrderTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
         configBuilder.tab("conditionalOrder", "Conditional Order")
             .forRoles(CASE_WORKER, LEGAL_ADVISOR, SOLICITOR, SUPER_USER)
-            .showCondition("coDateSubmitted=\"*\"")
+            .showCondition("coApplicant1SubmittedDate=\"*\"")
```

Comment:
theoretically this needs to be more complex now as with a joint it's the latter date of app1 or app2, but we'll have to tackle it later

Reply:
do we show the tab if either of them are present (for joint) or both?

Reply:
if joint then show oldest of (app1 / app2 submission date)
if sole just show app1

Reply:
too complicated for now I think